test/test_ops.py

```diff
@@ -920,6 +957,14 @@ def test():
         xfail('as_strided'),
         xfail('nn.functional.fractional_max_pool2d'),
         skip('solve'),
+
+        # Issue with newer svd implementation and forwardAD
+        # https://github.com/pytorch/pytorch/pull/69827
+        # https://github.com/pytorch/pytorch/pull/70253
+        #
+        # RuntimeError: tensor.mH is only supported on matrices or batches of matrices. Got 1-D tensor.
+        xfail('linalg.cond'),
+        xfail('linalg.svdvals'),
```

Comment:
(just taking notes) this is a regression

Reply:
This one was working before but throwing a warning saying that "we're defaulting to an iterative method note that this is inefficient" or smth like that. Note that `linalg.svdvals` did not and does not have an unpacking rule before, and we have not changed its implementation anyway, so it's not clear to me why it was falling back to an iterative method that made it work before and now it is not. Any ideas?

Reply:
`xfail('linalg.svdvals')` was moved from line 910 to 967, maybe not a regression as once we started decomposing `linalg_svdvals` it was directly xfailed.